quickwit/quickwit-metastore/test-data/index-metadata/v0.5-48d02d9017e35d54e3c67cf2e5d3002a.expected.json

```diff
@@ -0,0 +1,109 @@
+{
+  "checkpoint": {
+    "kafka-source": {
+      "00000000000000000000": "00000000000000000042"
+    }
+  },
```

Comment:
we can remove this file

Reply:
Not this one but we another one that was also created. This one keeps getting recreated.